releasenotes/notes/add-graphql-core-support-fd20dcbebfcaa8c7.yaml

```diff
@@ -0,0 +1,5 @@
+---
+features:
+  - |
+    Add tracing support for ``graphql-core``. Version 2.0+ is fully
+    supported. By transitive dependencies graphene>2.0 is also supported.
```

Comment:
hmm I'm on the fence about mentioning graphene here.

If we do say that we support it then it should be tested. Maybe we follow up with "graphene" support even if it's just tests? WDYT?

Reply:
Yup I'll leave it out